### ISSUE TITLE
Could not load "D:\Development\PHP-Unit-Test\phpunit.xml".

### ISSUE THREAD

**Issue by noncent** [evgeny-golubev/SimplePHPUnit-for-Sublime-Text#5]
Hi, First of all Thanks to create a lovely plugins for ST3, I am trying to use SimplePHPUnit. I installed on my ST3. When I am trying to run PHPUnit it's giving me error 'Could not load "D:\Development\PHP-Unit-Test\phpunit.xml".' I have created phpunit.xml in root directory. Here is error:

Could not load "D:\Development\PHP-Unit-Test\phpunit.xml".
[Finished in 1.2s with exit code 1]
[cmd: phpunit --stderr]
[dir: D:\Development\PHP-Unit-Test]
[path: C:\Program Files (x86)\Notepad++\plugins\database;C:\Program Files (x86)\Notepad++\plugins\PHP\ext;C:\Program Files (x86)\Notepad++\plugins\PHP;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Python27\pythonw.exe;C:\Program Files (x86)\nodejs\;D:\wamp\bin\mysql\mysql5.5.24\bin\;C:\Python27;C:\Python27\Lib\site-packages\;C:\Python27\Scripts\;C:\Program Files (x86)\Common Files\GTK\2.0\bin;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\TortoiseSVN\bin;D:\wamp\bin\php\php5.3.13;C:\ProgramData\Composer\bin;C:\OpenSSL-Win32\bin;C:\Program Files (x86)\MySQL\MySQL Utilities\;C:\ffmpeg\bin;D:\wamp\bin\php\php5.3.13]

**Comment by evgeny-golubev** [comment]
Sorry, your problem is still relevant?

**Comment by atty31** [comment]
yes it is relevant for me. I got the same issue: phpunit.xml or phpunit.xml.dist not found
